### PR TITLE
New CIC uniform

### PR DESCRIPTION
**Pull request by CapCamIII**
# About the pull request

Does new CIC sprites, they look cool.
New cosmetic options with brown gloves and boots
New outfit that looks like the operation uniform gorman wears in the movie
CTs getting actual caps instead of beanies is nice, RO(i'm not calling it QM 🖕) still has a distinct hat to seperate the two out.

# Explain why it's good for the game

New sprites, service uniforms being static to the map makes sense as the operations uniform is what is wore groundside in the movie, service uniform is more of a shipside/non-combat uniform

# Images

image

# Changelog
:cl: CapCamIII, AmoryBlaine
add: Adds new service uniform sprite that is standard across maps, sprites by AmoryBlaine
add: Resprites old dress uniform, renames it, adds to XO vendor as a uniform choice as it fits there now, sprites by AmoryBlaine
add: Adds new uniform, command boiler suit like gorman wears in the movie, available in SO locker, SO vendor, XO vendor, CO vendor, sprites by AmoryBlaine
add: Adds new brown versions of gloves and boot to custom loadout, also lowers the price of jungle boots to two, sprites by AmoryBlaine
add: Seperates officer cap and RO cap, officers get officer cap, QM gets RO cap, CTs get req cap, sprites once again by AmoryBlaine
add: UPP boots and gloves use the new brown sprite, as it fits their color scheme and looks cool.
fix: Freelancer leader now spawns with a knife in their boots
imageadd: Resprites patrol cap, officer cap, SEA cap, standard black boot, standard black glove, and formal gloves, sprites by AmoryBlaine
/:cl:

### CODE REVIEW

File: code/game/machinery/vending/vendor_types/crew/senior_officers.dm [hunk 1]
```diff
@@ -173,9 +173,12 @@ GLOBAL_LIST_INIT(cm_vending_clothing_req_officer, list(
 		list("Quartermaster Uniform", 0, /obj/item/clothing/under/rank/qm_suit, MARINE_CAN_BUY_UNIFORM, VENDOR_ITEM_MANDATORY),
 		list("Headset", 0, /obj/item/device/radio/headset/almayer/qm, MARINE_CAN_BUY_EAR, VENDOR_ITEM_MANDATORY),
 		list("Satchel", 0, /obj/item/storage/backpack/marine/satchel/tech, MARINE_CAN_BUY_BACKPACK, VENDOR_ITEM_MANDATORY),
-		list("Req Cap", 0, /obj/item/clothing/head/cmcap/req, MARINE_CAN_BUY_MASK, VENDOR_ITEM_MANDATORY),
 		list("Quartermaster Jacket", 0, /obj/item/clothing/suit/storage/RO, MARINE_CAN_BUY_MRE, VENDOR_ITEM_MANDATORY),
 
+		list("Headgear (CHOOSE 1)", 0, null, null, null),
+		list("Quartermaster Cap", 0, /obj/item/clothing/head/cmcap/req/ro, MARINE_CAN_BUY_MASK, VENDOR_ITEM_RECOMMENDED),
+		list("Requsitions Cap", 0, /obj/item/clothing/head/cmcap/req, MARINE_CAN_BUY_MASK, VENDOR_ITEM_REGULAR),
```
Comment (morrowwolf) [inline code/game/machinery/vending/vendor_types/crew/senior_officers.dm:180]: Requisitions

**Review by morrowwolf — CHANGES_REQUESTED** [review]

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by kooarbiter** [comment]
I can't believe nobody's discussing the huge balance change of giving one random ert role a bootknife smh smh

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by nauticall — COMMENTED** [review]
As much as everyone likes to talk about movie accuracy, this is one of the instances where I think an exception should be made. The boiler suits, while they look good, don't represent the idea of an "officer" at all, and if the star wasn't there, people would easily confuse SOs for being maint/cargo techs or janitors. The existing dress shirt + slacks combination, while a bit uptight for a ship's uniform, does the job of distinguishing the SO as an officer and someone to actually at least _try_ to take seriously.

I'm not saying to scrap these boiler uniforms entirely, but they should not be used as staff officer uniforms, sorry. People are not gonna look at them and think, "oh, an officer". If they don't want to look like an officer then they can just wear regular fatigues.

**Review by nauticall — CHANGES_REQUESTED** [review]
image
White uniform lacks contrast here and conflicts with existing styles on whites. Not saying to blacken the edges completely, but make the outlines _slightly_ darker.

image
We already have dress blues, they just haven't been added to uniform vendors yet. Making yet another 'dress' uniform for marines will just add to confusion and redundancy.

image
Boots are too dark. Make them match the brown gloves in palette.

image
Palette for the gloves here are all over the place. You'd be better off just having it use the same sprite as the latex gloves, honestly, or at least basing the sprite off those. Both latex and formal cloth gloves are white and smooth-textured.

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Comment by LC4492** [comment]
As an idea, I think it would be nice if the Chief Engineer also received a more..."tailored" outfift, of which the beret matched with a possible officer jacket, but this is already good as it is

**Comment by cm13-github** [comment]
This pull request has conflicts, please resolve those before we can evaluate the pull request.

**Comment by cm13-github** [comment]
Conflicts have been resolved. A maintainer will review the pull request shortly.

**Review by nauticall — APPROVED** [review]
👌 sprites good now, cheers

**Comment by morrowwolf** [comment]
TM'ing just in case of any niche sprite issues

**Pull request merged**